bundles/org.openhab.core.automation.rest/src/main/java/org/openhab/core/automation/rest/internal/ThingActionsResource.java

```diff
@@ -171,11 +179,24 @@ public Response getActions(@PathParam("thingUID") @Parameter(description = "thin
                     continue;
                 }
 
+                List<ConfigDescriptionParameter> inputParameters = new ArrayList<>();
+                for (Input input : actionType.getInputs()) {
+                    ConfigDescriptionParameter parameter = convertToConfigDescriptionParameter(input);
+                    if (parameter != null) {
+                        inputParameters.add(parameter);
+                    } else {
+                        inputParameters = null;
+                        break;
+                    }
+                }
+
                 ThingActionDTO actionDTO = new ThingActionDTO();
                 actionDTO.actionUid = actionType.getUID();
                 actionDTO.description = actionType.getDescription();
                 actionDTO.label = actionType.getLabel();
                 actionDTO.inputs = actionType.getInputs();
+                actionDTO.inputsAsConfigParameters = inputParameters == null ? null
```

Comment:
I would rather name it:

```suggestion
                actionDTO.inputConfigDescriptions = inputParameters == null ? null
```

Reply:
Ok if you prefer.

Reply:
Sorry for picking this naming topic up again, but would it make sense to use the singular?
inputConfigDescription?

Reply:
I would rather suggest inputsConfigDescription. The plural will indicate we have a list of something.

Or maybe configDescriptionInputs ?